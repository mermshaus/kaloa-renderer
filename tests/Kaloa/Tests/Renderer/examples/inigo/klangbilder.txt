Dies könnte der Auftakt zu einer losen Serie sein, in der ich auf frei herunterladbare Musik hinweise, die ich irgendwo aufgeschnappt habe und die mir aus mehr oder weniger nachvollziehbaren Gründen hörenswert erscheint.

Ich weiß noch nicht, ob ich an dieser Form festhalten werde. Auf den ersten Blick erscheint es mir praktischer, die Einträge sofort einzeln einzustellen, statt zu warten, bis sich ausreichend Material für einen eigenen Beitrag angesammelt hat. Aktuell konnte ich jedoch auf ein entsprechend großes Archiv zurückgreifen.

[img=example.png]Bildunterschrift[/img]

Inhaltlicher Zusammenhang der Einträge oder Trennungen nach Genre sind übrigens vorerst nicht Teil des Konzepts. Große Ahnung hätte ich davon ohnehin nicht. Zudem werde ich nach Möglichkeit Downloadseiten verlinken, nicht direkt die Dateien. Das gehört sich so.

[url=http://www.shauninman.com/archive/2006/09/17/the_lovely_life_ep]Shaun Inman: The Lovely Life EP[/url] und [url=http://www.shauninman.com/archive/2008/07/10/lot_ep]Lot EP[/url]. Shaun Inmans Musik ist ruhig, dabei aber energetisch, getragen von gleichmäßigem Gesang, Gitarrenspiel und sparsam eingesetzten elektronischen Effekten. Sie drängt sich angenehm in den Hintergrund.

[url=http://www.minilogue.com/downloads_music.htm]Minilogue: Liveset Berlin[/url]. 83 Minuten eleganter Minimaltechno aus Schweden. Gute elektronische Musik spielt geschickt mit Verzögerungselementen und der dadurch beim Zuhörer erzeugten Erwartungshaltung. Die Reaktionen des Publikums machen einen großen Teil des Reizes dieses Mitschnitts aus.

[url=http://www.archive.org/details/etree]Internet Archive: Live Music Archive[/url]. Eine Sammlung qualitativ hochwertiger Liveaufnahmen [url=http://www.archive.org/browse.php?collection=etree&field=%2Fmetadata%2Fcreator]verschiedener Künstler[/url]. Die meisten Interpreten sind (mir) unbekannt, wärmstens empfehlen kann ich allerdings die "Tucson, Arizona"-Fraktion mit [url=http://www.archive.org/details/gsand2005-09-02.flac16]Giant Sand[/url] und natürlich [url=http://www.archive.org/details/calexico2003-04-25.flac16]Calexico[/url].

[url=http://jackconte.googlepages.com/]Jack Conte: VideoSongs[/url]. Der Witz an der Sache ist, dass hier nicht einfach zwei Songs gemischt werden, sondern Jack Conte jeden Ton irgendwann selbst eingespielt oder eingesungen hat. Genauere Angaben dazu etwa im Infobereich eines der zugehörigen [url=http://youtube.com/watch?v=0X-PRpqj7N4]Videos[/url].

[url=http://www.nytimes.com/2008/04/23/arts/music/23aria.html]Juan Diego Flórez: La Fille du Régiment: Ah! Mes Amis[/url]. Der Tenor Juan Diego Flórez singt fehlerfrei neun hohe Cs hintereinander, bekommt stehende Ovationen, setzt ein kompliziertes Signalsystem in Gang und singt die erste Solo-Zugabe in der Metropolitan Opera in New York seit 1994. Eine nette Geschichte und eine Aufnahme, die man gut beim Zähneputzen hören kann.

[url=http://www.partsandlabor.net/av.html]Parts & Labor[/url]. Parts & Labor verbreiten Hektik mit rohem Elektrorock. Der Schlagzeuger muss vermutlich alle zwei Songs ausgewechselt werden. Leider nicht als Download angeboten: "[url=http://youtube.com/watch?v=YwCFP1qyz9k]The Gold We're Digging[/url]".
