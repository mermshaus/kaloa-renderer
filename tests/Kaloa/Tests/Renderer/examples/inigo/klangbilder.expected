<p>Dies könnte der Auftakt zu einer losen Serie sein, in der ich auf frei herunterladbare Musik hinweise, die ich irgendwo aufgeschnappt habe und die mir aus mehr oder weniger nachvollziehbaren Gründen hörenswert erscheint.</p>

<p>Ich weiß noch nicht, ob ich an dieser Form festhalten werde. Auf den ersten Blick erscheint es mir praktischer, die Einträge sofort einzeln einzustellen, statt zu warten, bis sich ausreichend Material für einen eigenen Beitrag angesammelt hat. Aktuell konnte ich jedoch auf ein entsprechend großes Archiv zurückgreifen.</p>
<div class="img">
  <div class="img_left" style="width: 408px;">
    <div class="img_border">
      <img src="/home/public_html/marc/kaloa-renderer/tests/Kaloa/Tests/Renderer/examples/inigo/klangbilder/example.png" alt="Bild: /home/public_html/marc/kaloa-renderer/tests/Kaloa/Tests/Renderer/examples/inigo/klangbilder/example.png" title="/home/public_html/marc/kaloa-renderer/tests/Kaloa/Tests/Renderer/examples/inigo/klangbilder/example.png" width="400" height="300" />
    </div>
    <div class="img_content">Bildunterschrift</div>
  </div>
</div>

<p>Inhaltlicher Zusammenhang der Einträge oder Trennungen nach Genre sind übrigens vorerst nicht Teil des Konzepts. Große Ahnung hätte ich davon ohnehin nicht. Zudem werde ich nach Möglichkeit Downloadseiten verlinken, nicht direkt die Dateien. Das gehört sich so.</p>

<p><a href="http://www.shauninman.com/archive/2006/09/17/the_lovely_life_ep" title="Open &quot;http://www.shauninman.com/archive/2006/09/17/the_lovely_life_ep&quot;">Shaun Inman: The Lovely Life EP</a> und <a href="http://www.shauninman.com/archive/2008/07/10/lot_ep" title="Open &quot;http://www.shauninman.com/archive/2008/07/10/lot_ep&quot;">Lot EP</a>. Shaun Inmans Musik ist ruhig, dabei aber energetisch, getragen von gleichmäßigem Gesang, Gitarrenspiel und sparsam eingesetzten elektronischen Effekten. Sie drängt sich angenehm in den Hintergrund.</p>

<p><a href="http://www.minilogue.com/downloads_music.htm" title="Open &quot;http://www.minilogue.com/downloads_music.htm&quot;">Minilogue: Liveset Berlin</a>. 83 Minuten eleganter Minimaltechno aus Schweden. Gute elektronische Musik spielt geschickt mit Verzögerungselementen und der dadurch beim Zuhörer erzeugten Erwartungshaltung. Die Reaktionen des Publikums machen einen großen Teil des Reizes dieses Mitschnitts aus.</p>

<p><a href="http://www.archive.org/details/etree" title="Open &quot;http://www.archive.org/details/etree&quot;">Internet Archive: Live Music Archive</a>. Eine Sammlung qualitativ hochwertiger Liveaufnahmen <a href="http://www.archive.org/browse.php?collection=etree&amp;field=%2Fmetadata%2Fcreator" title="Open &quot;http://www.archive.org/browse.php?collection=etree&amp;field=%2Fmetadata%2Fcreator&quot;">verschiedener Künstler</a>. Die meisten Interpreten sind (mir) unbekannt, wärmstens empfehlen kann ich allerdings die "Tucson, Arizona"-Fraktion mit <a href="http://www.archive.org/details/gsand2005-09-02.flac16" title="Open &quot;http://www.archive.org/details/gsand2005-09-02.flac16&quot;">Giant Sand</a> und natürlich <a href="http://www.archive.org/details/calexico2003-04-25.flac16" title="Open &quot;http://www.archive.org/details/calexico2003-04-25.flac16&quot;">Calexico</a>.</p>

<p><a href="http://jackconte.googlepages.com/" title="Open &quot;http://jackconte.googlepages.com/&quot;">Jack Conte: VideoSongs</a>. Der Witz an der Sache ist, dass hier nicht einfach zwei Songs gemischt werden, sondern Jack Conte jeden Ton irgendwann selbst eingespielt oder eingesungen hat. Genauere Angaben dazu etwa im Infobereich eines der zugehörigen <a href="http://youtube.com/watch?v=0X-PRpqj7N4" title="Open &quot;http://youtube.com/watch?v=0X-PRpqj7N4&quot;">Videos</a>.</p>

<p><a href="http://www.nytimes.com/2008/04/23/arts/music/23aria.html" title="Open &quot;http://www.nytimes.com/2008/04/23/arts/music/23aria.html&quot;">Juan Diego Flórez: La Fille du Régiment: Ah! Mes Amis</a>. Der Tenor Juan Diego Flórez singt fehlerfrei neun hohe Cs hintereinander, bekommt stehende Ovationen, setzt ein kompliziertes Signalsystem in Gang und singt die erste Solo-Zugabe in der Metropolitan Opera in New York seit 1994. Eine nette Geschichte und eine Aufnahme, die man gut beim Zähneputzen hören kann.</p>

<p><a href="http://www.partsandlabor.net/av.html" title="Open &quot;http://www.partsandlabor.net/av.html&quot;">Parts & Labor</a>. Parts & Labor verbreiten Hektik mit rohem Elektrorock. Der Schlagzeuger muss vermutlich alle zwei Songs ausgewechselt werden. Leider nicht als Download angeboten: "<a href="http://youtube.com/watch?v=YwCFP1qyz9k" title="Open &quot;http://youtube.com/watch?v=YwCFP1qyz9k&quot;">The Gold We're Digging</a>".</p>